torchaudio/functional.py

```diff
@@ -151,7 +152,8 @@ def istft(
     assert 0 < win_length <= n_fft
 
     if window is None:
-        window = torch.ones(win_length, requires_grad=False, device=device, dtype=dtype)
```

Comment:
nit: indentation should be 4 spaces -- I recommend running the code in flake8 to catch those typos :)

Reply:
ah I have been having a problem with my coc.nvim settings recently--thanks for pointing it out!